src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -120,34 +119,6 @@
     </ItemGroup>
   </Target>
 
-  <!--
-    Deploy a net472 version of the BuildHost process.
-  -->
-  <Target Name="DeployNetFrameworkBuildHost" AfterTargets="ResolveProjectReferences" Condition="'$(DesignTimeBuild)' != 'true'">
```

Comment:
This all moves to the MSBuildWorkspace project directly. You'll see this oevr there (with some changes to make NuGet happier.)